.github/workflows/benchmark-pull-request.yml

```diff
@@ -109,7 +109,7 @@ jobs:
           echo "prHeadRefSha=$headRefSha" >> $GITHUB_ENV
       - id: get_approvers
```

Comment:
Looks like we are trying to solve this problem in multiple ways, I'd prefer this API based method:
https://github.com/opensearch-project/OpenSearch/blob/6e27e5ad8e37944605bef61e14f1972f440e4724/.github/workflows/maintainer-approval.yml#L11-L32

Reply:
Thanks @peternied, I think this is the better approach.
I double checked https://github.com/opensearch-project/OpenSearch/pull/15163 and it seems like we aren't adding all the maintainers to https://github.com/opensearch-project/OpenSearch/blob/6e27e5ad8e37944605bef61e14f1972f440e4724/.github/CODEOWNERS#L37 either

Reply:
This looks way better, will update it.

Reply:
@peternied Updated as per recommendation.